holonix/README.md

```diff
@@ -0,0 +1,128 @@
+# Holonix
+
+NixOS and `nix-shell` in other OS environments implements the HC ops tooling.
+
+## Scope
+
+Where `nix-shell` is supported Holonix is the standard approach for:
+
+- Installing dependencies
+- Running tests
+- Building wasm
+- Building binaries
+- Implementing CI scripts
+- Conductor management
+- Installing situational tooling (e.g. `cargo-edit` for versioning)
+- Release management, automation and deploying official binaries
+- BAU automation and reporting tasks**
+- Downloading prebuilt official binaries from github**
+- Managing and updating binaries across official releases**
+
+** Coming Soon!
+
+### Supported environments
+
+NixOS is a dedicated linux distro and so is standalone by its nature.
+
+`nix-shell` is a cli tool supported across Mac OS and Linux distros.
+
+https://nixos.org/nix/download.html
+
+There is theoretical support for `nix-shell` in Windows Subsystem Linux (WSL).
+WSL support is currently broken upstream https://github.com/NixOS/nix/issues/1203
+
+**Use `nix-shell` if you are on anything other than Windows!**
+
+### Why nix?
+
+Nix approach offers unique benefits:
+
+- Dependencies are injected into a single shell session only
+  - Minimal modifications to the host environment
+  - No need to maintain/rerun/troubleshoot installation scripts
+  - Further isolation from host environment can be achieved with `nix-shell --pure`
+- Dependencies are hashed
+  - "Dependency hell" is avoided
+  - Nice parallels with the hashed zomes model
+  - Security + reliability benefits
+- Dependencies can be garbage collected with the `nix-collect-garbage` command
+- Single "package manager" across most operating systems
+- Ability to ship utility scripts in the `shell.nix` file
+- Access to the nix functional programming language for dependencies/script management
+  - Allows for a structured approach to coding the ops infrastructure
+  - Modern programming paradigm such as immutability, scopes, data structures, etc.
+- NixOS runs on HoloPorts so `nix-shell` provides similar behaviour/environment
+  - Everything is pinned to the same nix channel and commit
+
+And other benefits:
+
+- Active and helpful community of contributors
+  - IRC at #nixos
+  - wiki at https://nixos.wiki/
+- Ability to bundle scripts into convenient binaries for daily tasks
+- Nice CLI interface with many flexible, well documented configuration options
+- Available in shebang form as `#! /usr/bin/env nix-shell`
+  - http://iam.travishartwell.net/2015/06/17/nix-shell-shebang/
+
+## Installation
+
+Follow the instructions to install `nix-shell` from:
+
+https://nixos.org/nix/download.html
+
+Once installed run `nix-shell` from the `holochain-rust` repository root.
+The `nix-shell` command will detect the `shell.nix` file and build everything.
+
+The first run will be slow as nix downloads, builds and caches dependencies.
+Everything is downloaded to `/nix/store/...` and re-used on subsequent runs.
+
+## Contributing
```

Comment:
I think this contributing section could also use a bit of instruction on the `dist` folders which it looks like are used to hold derivations/scripts for build artifacts for release.  Correct?

Reply:
this isn't "contributing to holochain core" it is "contributing to holonix"